src/dragonball/src/dbs_pci/src/vfio.rs

```diff
@@ -884,7 +884,7 @@ impl Region {
     }
 }
 
-struct VfioPciDeviceState<C: PciSystemContext> {
+pub struct VfioPciDeviceState<C: PciSystemContext> {
```

Comment:
Why make state a pub struct?

Reply:
cause we need to get the state of `VfioPciDevice`, if we make this pub, Rust will complain that we leak a private struct. Error is attached.
![image](https://github.com/kata-containers/kata-containers/assets/31121654/876d7042-dd81-40c1-918f-ff261b077f4b)